src/helpers/useUncomplitedTodos.ts

```diff
@@ -0,0 +1,5 @@
+import { Todo } from '../types/Todo';
+
+export const useUncompletedTodos = (todos: Todo[]) => (
```

Comment:
The same regarding naming:
```suggestion
export const getActiveTodos = (todos: Todo[]) => (
```

Reply:
Cause that warn from linter:
![image](https://github.com/mate-academy/react_todo-app-with-api/assets/67290782/c086af70-5317-4425-9870-6478caad30b6)

Reply:
i reimplement that.